ppr-ui/tests/unit/SecuredParties.spec.ts

```diff
@@ -201,4 +251,34 @@ describe('Secured party amendment tests', () => {
     expect(item3.querySelectorAll('td')[4].textContent).toContain('Undo')
   })
 
+  it('displays the error', async () => {
+    const parties = cloneDeep(mockedSecuredParties1)
+    expect(parties.length).toEqual(1)
+    parties[0].action = ActionTypes.REMOVED
+    wrapper.vm.$data.securedParties = parties
+    wrapper.vm.$props.setShowInvalid = true
+    expect(wrapper.vm.getSecuredPartyValidity()).toBe(false)
+    wrapper.vm.$data.showErrorSecuredParties = true
+    await Vue.nextTick()
+    expect(wrapper.findAll('.invalid-message').length).toBe(1)
+  })
+
+
+  it('goes from valid to invalid', async () => {
+    const parties = cloneDeep(mockedSecuredParties1)
+    expect(parties.length).toEqual(1)
+    wrapper.vm.$data.securedParties = parties
+    await Vue.nextTick()
+    expect(wrapper.vm.getSecuredPartyValidity()).toBe(true)
+    // remove said secured party
+    // click the drop down arrow
+    wrapper.find('.v-data-table .party-row .actions__more-actions__btn').trigger('click')
+    await Vue.nextTick()
+    //click remove
+    wrapper.find('.actions__more-actions .v-list-item__subtitle').trigger('click')
+    await Vue.nextTick()
+    expect(wrapper.vm.getSecuredPartyValidity()).toBe(false)
```

Comment:
can you please update this to include ```expect(wrapper.findAll('.invalid-message').length).toBe(0)``` before you click the remove btn and then ```expect(wrapper.findAll('.invalid-message').length).toBe(1)``` after? It should fail right now due to my above comment, but it is the desired functionality I think

Reply:
added tests